packages/components/components/elvis-stepper/src/react/styledComponents.tsx

```diff
@@ -39,6 +38,9 @@ export const Steps = styled.div<StepsProps>`
   align-items: ${({ $type }) => ($type === 'vertical' ? 'start' : 'center')};
   justify-content: center;
   padding: unset;
+  width: 100%;
+  max-width: ${({ $type }) => ($type === 'vertical' ? 'initial' : '400px')};
```

Comment:
This is not something you need to change, but in Styled Components, if you do it like this then no style will be shipped for the cases where it results in `undefined`.
```suggestion
  max-width: ${({ $type }) => $type === 'vertical' && '400px')};
```
Basically, you can use short-circuiting safely 😄 
![image](https://github.com/3lvia/designsystem/assets/38653456/42ceaf12-c8f1-4149-975e-f9675f3afeec)
https://styled-components.com/docs/advanced